ansible/playbooks/deploy_moriod.yml

```diff
@@ -0,0 +1,118 @@
+---
+- name: Provision EC2 Instances and Configure DNS
+  hosts: localhost
+  gather_facts: no
+  collections:
+    - amazon.aws
+    - community.general
+  vars:
+    region: "eu-central-1"
+    subnet_id: "subnet-04faec9c5fe410b3e"
+    security_group_id: "sg-09a11726f166660ca"
+    zone_id: "Z04114953S6ELFC6IJG0T"  # Replace with your Route 53 hosted zone ID
+    ami_id: "ami-0552f4006d8da5f5f"  # Replace with your preferred AMI ID
+
+  tasks:
+    - name: Generate random DNS names for instances
+      set_fact:
+        dns_names: []
+        
+    - name: Generate random DNS names and append to list
+      set_fact:
+        dns_names: "{{ dns_names + [lookup('community.general.random_string', count=1, length=3, special=False, upper=False)] }}"
+      loop: "{{ range(node_count)|list }}"
+
+    - name: Create an SSH Key Pair
+      amazon.aws.ec2_key:
+        name: "morio_test_{{ pr_id }}-{{ item }}"
```

Comment:
Please use `morio_itest_....` here too, instead of `morio_test....`

Reply:
Corrected.